HHS/Head-Start-TTADP
# [Production] Add state filter, remove mismatched global queries

## Description of change

- The state filter has been added to the filter panel on the landing page and the regional dashboard
- Previously, there were two places in the code where we specifically excluded reports with start dates before the creation of the hub. A slight discrepancy in those implementations led to numbers not agreeing across the hub. All reports before that date have been removed, and the code filtering that out has also been removed, so we won't have a situation like that again.

## How to test

- Just try and find a report before 9/1, I bet you can't
- Create a state filter on either page. Ensure it works properly.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-534
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-442


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
